mon-pix/app/router.js

```diff
@@ -59,12 +59,11 @@ Router.map(function() {
       this.route('send-profile', { path: '/envoi-profil' });
       this.route('profile-already-shared', { path: '/deja-envoye' });
     });
-  });
-
-  this.route('assessment-campaigns', { path: '/campagnes/:campaign_code/evaluation' }, function() {
-    this.route('start-or-resume', { path: '/' });
-    this.route('tutorial', { path: '/didacticiel' });
-    this.route('skill-review', { path: '/resultats/:assessment_id' });
+    this.route('evaluation', function() {
```

Comment:
```suggestion
    this.route('assessment', { path: 'evaluation' }, function() {
```
le type de campagne c'est bien `ASSESSMENT`, et la traduction c'est évaluation